src/core/src/node.cpp

```diff
@@ -74,7 +74,7 @@ ov::Node::Node(const OutputVector& arguments, size_t output_size) : Node() {
     set_output_size(output_size);
 }
 
-ov::Node::~Node() {
+ov::Node::~Node() noexcept(false) {
```

Comment:
Destructor cannot throw exceptions

PR with the same changes: https://github.com/openvinotoolkit/openvino/pull/10123

Reply:
Reverted